Packs/ServiceDeskPlus/Integrations/ServiceDeskPlus/ServiceDeskPlus.py

```diff
@@ -820,18 +832,29 @@ def generate_refresh_token(client: Client, args: Dict) -> Tuple[str, dict, Any]:
 
 def main():
     params = demisto.params()
-    server_url = SERVER_URL[params.get('server_url')]
-
-    client = Client(url=server_url + API_VERSION,
-                    use_ssl=not params.get('insecure', False),
-                    use_proxy=params.get('proxy', False),
-                    client_id=params.get('client_id'),
-                    client_secret=params.get('client_secret'),
-                    refresh_token=params.get('refresh_token'),
-                    fetch_time=params.get('fetch_time') if params.get('fetch_time') else '7 days',
-                    fetch_status=params.get('fetch_status'),
-                    fetch_limit=int(params.get('fetch_limit')) if params.get('fetch_limit') else 50,
-                    fetch_filter=params.get('fetch_filter') if params.get('fetch_filter') else '')
+    server_url = params.get('server_url')
+    if server_url == 'On-Premise':
```

Comment:
Can you explain how the url will be 'On-Premise'?

Reply:
It will be one of the options in the `Data Center Location` param

<img width="857" alt="Screen Shot 2021-04-07 at 18 44 04" src="https://user-images.githubusercontent.com/54398957/113895335-8e856d00-97d1-11eb-8337-2d75a5fb5920.png">